README.md

```diff
@@ -67,6 +67,7 @@
 24. [Keras in Motion video course](https://www.manning.com/livevideo/keras-in-motion)
 25. [Practical Deep Learning For Coders](http://course.fast.ai/) by Jeremy Howard - Fast.ai
 26. [Introduction to Deep Learning](http://deeplearning.cs.cmu.edu/) by Prof. Bhiksha Raj (2017)
+27. [Machine Learning Crash Course with TensorFlow APIs](https://developers.google.com/machine-learning/crash-course/?utm_source=google-ai&utm_medium=card-image&utm_campaign=training-hub&utm_content=ml-crash-course) -Google AI
```

Comment:
It could be simplified to https://developers.google.com/machine-learning/crash-course/

Reply:
Sure it can be